features/step_definitions/login_to_KFS.rb

```diff
@@ -1,37 +1,39 @@
+include KsbClientUtilities
+
 Given /^I am logged in as a KFS Operations$/ do
-  perform_backdoor_login(get_first_principal_name_for_role('KFS-SYS', 'Operations'))
+  perform_backdoor_login(get_random_kfs_operations_user)
 end
 
 Given /^I am logged in as a KFS Fiscal Officer$/ do
   perform_backdoor_login(get_aft_parameter_value(ParameterConstants::DEFAULT_FISCAL_OFFICER))
 end
 
 Given /^I am logged in as a KFS User$/ do
-  perform_backdoor_login(get_random_principal_name_for_role('KFS-SYS', 'User'))
+  perform_backdoor_login(get_random_kfs_user)
```

Comment:
Nancy helped me offline to find the ksb_client_utilities.rb and the new method. Thank you Nancy.